components/terrain/compositemaprenderer.cpp

```diff
@@ -59,26 +59,29 @@ void CompositeMapRenderer::drawImplementation(osg::RenderInfo &renderInfo) const
 
     while (!mImmediateCompileSet.empty())
     {
-        CompositeMap* node = *mImmediateCompileSet.begin();
+        osg::ref_ptr<CompositeMap> node = *mImmediateCompileSet.begin();
         mCompiled.insert(node);
```

Comment:
This insert seems to be useless. `mCompiled` is not used anywhere for reading.

Reply:
scrawl did not leave any comments, but it can be aimed just to keep references for handled nodes. Or it indeed can be redundant now.